tutorials/text/text_props.py

```diff
@@ -194,7 +194,15 @@
 # +------------------------------------------+--------------------------------+
 #
 #
-# which are lists of font names.
+# If any of generic family names appear in ``'font.family'``, we replace that entry
+# by all the entries in the corresponding rcParam mapping.
+# For example: ::
+#
+#    matplotlib.rcParams['font.family'] = ['MyFamily1', 'serif', 'MyFamily2']
+#    matplotlib.rcParams['font.serif'] = ['MySerif1', 'MySerif2']
```

Comment:
ditto, what is `MySerif1/2`?    Are these font names?  Where do I get these?  Where are the fonts installed?

Reply:
Same as above^^

(we could also say that all fonts installed in user's machine is a 'user-defined' fonts)

(also note that sometimes the word 'fonts' and 'font-families' are used interchangeably)